cli/src/pcluster/templates/login_nodes_stack.py

```diff
@@ -71,6 +71,54 @@ def _add_login_nodes_pool_launch_template(self):
                 subnet_id=self._pool.networking.subnet_ids[0],
             )
         ]
+
```

Comment:
Let's maintain these scripts outside of this PR (since we'll manage them in the cookbook package).
Not using user_data.

Reply:
Done